anaconda

```diff
@@ -1139,7 +1139,7 @@ if __name__ == "__main__":
             iutil.setenv("ftp_proxy", proxy.url)
             iutil.setenv("HTTPS_PROXY", proxy.url)
 
-    if flags.noverifyssl:
+    if flags.noverifyssl and hasattr(ksdata.method, "noverifyssl"):
```

Comment:
Please also add comment why the `noverifyssl` may not be present.

Reply:
Done